pkg/scte35/stream_test.go

```diff
@@ -0,0 +1,107 @@
+package scte35
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func AreEqualJSON(s1, s2 []byte) (bool, error) {
```

Comment:
helper functions shouldn't be exported. Also, possible to use https://pkg.go.dev/github.com/stretchr/testify/assert#JSONEq instead? We prefer testify (require/assert) for test validations.